pyproject.toml

```diff
@@ -0,0 +1,104 @@
+[tool.ruff]
+line-length = 100
+
+extend-exclude = [
+  # TODO: Enable verifying notebooks on subsequent iteration.
+  "*.ipynb",
+]
```

Comment:
Let's let ruff format the notebooks on a later iteration. Version 0.6 gained relevant capabilities.